doc/source/tutorial/stats/rv_infrastructure.md

```diff
@@ -0,0 +1,920 @@
+
+res_mps = optimize.minimize(nlps, x0, bounds=bounds)
+res_mps.x
+```
+
+For method of [L-moments](https://en.wikipedia.org/wiki/L-moment) (which attempts to match the distribution and sample L-moments):
+
+```{code-cell} ipython3
+def lmoment_residual(params):
+    c, scale = params
+    X = Weibull(c=c) * scale
+    E11 = stats.order_statistic(X, r=1, n=1).mean()
+    E12, E22 = stats.order_statistic(X, r=[1, 2], n=2).mean()
+    lmoments_X = [E11, 0.5*(E22 - E12)]  # the first two l-moments of the distribution
+    lmoments_x = stats.lmoment(data, order=[1, 2])  # first two l-moments of the data
+    return np.linalg.norm(lmoments_x - lmoments_X)  # Minimize the norm of the difference
+
+x0 = [0.4, 3]  # This method is a bit sensitive to the initial guess
+res_lmom = optimize.minimize(lmoment_residual, x0, bounds=bounds)
+res_lmom.x, res_lmom.fun  # residual should be ~0
+```
+
+The plots for all three methods look similar and seem to describe the data, giving us some confidence that we have a good fit.
+
+```{code-cell} ipython3
+import matplotlib.pyplot as plt
+x = np.linspace(eps, 10, 300)
+
+for res, label in [(res_mle, "MLE"), (res_mps, "MPS"), (res_lmom, "L-moments")]:
+    c, scale = res.x
+    X = Weibull(c=c)*scale
+    plt.plot(x, X.pdf(x), '-', label=label)
+
+plt.hist(data, bins=np.linspace(0, 10, 30), density=True, alpha=0.1)
+plt.xlabel('x')
+plt.ylabel('pdf(x)')
+plt.legend()
+plt.ylim(0, 0.3)
+plt.show()
+```
+
+Estimation does not always involve fitting distributions to data. For instance, in gh-12134, a user needed to calculate the shape and scale parameters of the Weibull distribution to achieve a given mean and standard deviation. This fits easily into the same framework.
+
+```{code-cell} ipython3
+moments_0 = np.asarray([8, 20])  # desired mean and standard deviation
+def f(params):
+    c, scale = params
+    X = Weibull(c=c) * scale
+    moments_X = X.mean(), X.standard_deviation()
+    return np.linalg.norm(moments_X - moments_0)
+
+bounds.lb = np.asarray([0.1, 0.1])  # the Weibull distribution is problematic for very small c
+res = optimize.minimize(f, x0, bounds=bounds, method='slsqp')  # easily change the minimization method
+res
+```
+
+## New and Advanced Features
+
++++
+
+<a id='Transformations'></a>
+### Transformations
+
+Mathematical transformations can be applied to random variables. For instance, many elementary arithmetic operations (`+`, `-`, `*`, `/`, `**`) between random variables and scalars work.
+
+For instance, we can see that the reciprocal of a Weibull-distributed RV is distributed according to `scipy.stats.invweibull`. (An "inverse" distribution is typically the distribution underlying the reciprocal of a random variable.)
+
+```{code-cell} ipython3
+c = 10.6
+
+X = Weibull(c=10.6)  
+Y = 1 / X  # compare to `invweibull`
+Y.plot()
+
+x = np.linspace(0.8, 2.05, 300)
+plt.plot(x, stats.invweibull(c=c).pdf(x), '--')
+plt.show()
```

Comment:
The legends for the plots in this section are off (presumably should show both a red line and a dashed green line, rather than one blue one).

![image](https://github.com/user-attachments/assets/c13e67d9-bc60-4e3c-9115-b3a0f9cc79e0)

Reply:
Yeah, I'll play with it. Thanks.